kafka/client_async.py

```diff
@@ -681,8 +681,6 @@ def least_loaded_node(self):
         elif 'bootstrap' in self._conns:
             return 'bootstrap'
 
-        return None
```

Comment:
Slight preference for keeping this as an explicit return. Otherwise we still return None, it is just implicit when the function completes.

Reply:
No problem, reverted.